webhook.rst

```diff
@@ -77,27 +86,27 @@ The Webhook component routing needs to be defined:
             ;
         };
 
-Currently, the following third-party mailer services support webhooks:
+In this example, we are using ``mailer_mailgun`` as the webhook routing name.
+The routing name must be unique as this is what connects the provider with your
+webhook consumer code.
 
-=============== ==========================================
-Mailer service  Parser service name
-=============== ==========================================
-Mailgun         ``mailer.webhook.request_parser.mailgun``
-Postmark        ``mailer.webhook.request_parser.postmark``
-=============== ==========================================
-
-Set up the webhook in the third-party mailer. For Mailgun, you can do this
-in the control panel. As URL, make sure to use the ``/webhook/mailer_mailgun``
-path behind the domain you're using.
+The webhook routing name is part of the URL you need to configure at the
+third-party mailer provider. The URL is the concatenation of your domain name
+and the routing name you chose in the configuration (like
+``https://example.com/webhook/mailer_mailgun``.
 
-Mailgun will provide a secret for the webhook. Add this secret to your ``.env``
-file:
+For Mailgun, you will get a secret for the webhook. Store this secret as
+MAILER_MAILGUN_SECRET (in the :doc:`secrets management system
+</configuration/secrets>` or in a ``.env`` file).
 
-.. code-block:: env
+When done, add a :class:`Symfony\\Component\\RemoteEvent\\RemoteEvent` consumer
+to react to incoming webhooks (the webhook routing name is what connects your
+class to the provider).
 
-    MAILER_MAILGUN_SECRET=your_secret
-
-With this done, you can now add a RemoteEvent consumer to react to the webhooks::
+For mailer webhooks, react to the
+:class:`Symfony\\Component\\RemoteEvent\\Event\\Mailer\\MailerDeliveryEvent` or
```

Comment:
Having links to the events allows for a better discovery of all possible events.